src/book/01-team/04-history/02-past-members.mdx

```diff
@@ -5,3 +5,312 @@ title: Past Members
 description: Alumni and past members of the NUbots team.
 slug: /team/past-members
 ---
+
+The NUbots team has been fortunate to have many brilliant people working on it over the years. This page lists the members of the team in each year since our first RoboCup in 2002.
+
+This list may be incomplete, please let us know if we have missed someone who was a member of the team.
+
+---
+
+## 2021: Online
+
+**Academic Supervisors and Mentors:**
+
+- **Associate Professor Stephan Chalup**: Academic Head of the Team
+- **Dr. Alexandre Mendes**: Deputy Head of the Team
+- Dr. Joel Ferguson, Dr. Trent Houliston, Dr. Robert King, Professor Rick Middleton, Mr. Peter Turner
+
+**Student Members:**
+
+- **Josephus Paye II**: Team Leader
+- **Ysobel Sims**: Team Leader
+- Alex Biddulph, Peta Carlyle, Lachlan Court, Liam Craft, Nicholas Dzuira, Abigail Hall, Kip Hamiltons, Moosa Hassan, Yosiah de Koeyer, Sam McFarlane, Cameron Murtagh, Alana Noonan, Thomas O'Brien, Mikyla Peters, David Wieland, Joel Wong, Benjamin Young, Taylor Young
+
+**Supporting Members:**
+
+- Brendan Annable, Peter Salem, Cameron Steggles, Bryce Tuppurainen
+
+---
+
+## 2020: Cancelled
+
+RoboCup was cancelled in 2020 due to the COVID-19 pandemic.
+
+**Academic Supervisors and Mentors:**
+
+- **Associate Professor Stephan Chalup**: Academic Head of the Team
+- **Dr. Alexandre Mendes**: Deputy Head of the Team
+- Dr. Joel Ferguson, Dr. Trent Houliston, Dr. Robert King, Professor Rick Middleton, Mr. Peter Turner
+
+**Student Members:**
+
+- **Josephus Paye II**: Team Leader
+- **Ysobel Sims**: Team Leader
+- Alex Biddulph, Daniel Ginn, Shawn Hain, Kip Hamiltons, Timothy Mullen, Cameron Murtagh, David Wieland, Joel Wong, Taylor Young
+
+---
+
+## 2019: Sydney, Australia
+
+**Academic Supervisors and Mentors:**
+
+- **Associate Professor Stephan Chalup**: Academic Head of the Team
+- **Dr. Alexandre Mendes**: Academic Supervisor of the Team
+- Mr. Peter Turner, Dr. Robert King, Professor Rick Middleton, Dr. Trent Houliston
+
+**Student Members:**
+
+- **Alex Biddulph**: Team Leader
+- Matt Amos, Daniel Ginn, Shawn Hain, Timothy Mullen, Josephus Paye II, Ysobel Sims, Anita Sugo, David Wieland, Taylor Young
+
+**Supporting Members:**
+
+- Brendan Annable
+
+---
+
+## 2018: Montreal, Canada
+
+**Academic Supervisors and Mentors:**
+
+- **Associate Professor Stephan Chalup**: Academic Head of the Team
+- **Dr. Alexandre Mendes**: Academic Supervisor of the Team
+
+**Student Members:**
+
+- **Alex Biddulph**: Team Leader
+- Matt Amos, Daniel Ginn, Taylor Young
+
+**Supporting Members:**
+
+- Brendan Annable, Jade Fountain, Trent Houliston, Josephus Paye II, Ysobel Sims, David Wieland, Brandon Zahn
```

Comment:
```suggestion
- Brendan Annable, Jade Fountain, Trent Houliston, Josephus Paye II, Ysobel Sims, Brandon Zahn
```
I'm almost certain David joined the year after? iirc myself and Shawn were in the lab when he first came in.